radixdlt/src/main/java/com/radixdlt/middleware2/TokenFeeLedgerAtomChecker.java

```diff
@@ -129,26 +130,25 @@ private boolean isFeeGroup(ParticleGroup pg) {
 		List<SpunParticle> spunTransferableTokens = grouping.remove(TransferrableTokensParticle.class);
 		List<SpunParticle> spunUnallocatedTokens = grouping.remove(UnallocatedTokensParticle.class);
```

Comment:
Perhaps it would worth to use something like that:
`List<SpunParticle> spunTransferableTokens = Optional.ofNullable(grouping.remove(TransferrableTokensParticle.class)).orElseGet(List::of);`
This would simplify code downstream.